javav2/usecases/Creating_rds_item_tracker/Readme.md

```diff
@@ -2359,182 +2355,13 @@ The following JavaScript code represents the **contact_me.js** file that is used
-3. On the **Create New Application** page, enter the following values:
-   + **Application Name** - AWS Tracker
-   + **Description** - A description for the application
-
-![AWS Tracking Application](images/AWT6.png)
-
-4. Choose **Create**.
-5. Choose **Create a new environment**.
-6. Choose **Web server environment**.
-7. Choose **Select**.
-8. In the **Environment information** section, leave the default values.
-9. In the **Platform** section, choose **Managed platform**.
-10. For **Platform**, choose **Java** (accept the default values for the other fields).
-
-![AWS Tracking Application](images/AWT7.png)
-
-11. In the **Application code** section, choose **Upload your code**.
-12. Choose **Local file**, and then select **Choose file**. Browse to the JAR file that you created.  
-13. Choose **Create environment**. You'll see the application being created.
-
-![AWS Tracking Application](images/AWT8.png)
-
-When you’re done, you will see the application state the **Health** is **Ok** .
-
-![AWS Tracking Application](images/AWT9.png)
-
-14. To change the port that Spring Boot listens on, add an environment variable named **SERVER_PORT**, with the value **5000**.
-11. Add a variable named **AWS_ACCESS_KEY_ID**, and then specify your access key value.
-12. Add a variable named **AWS_SECRET_ACCESS_KEY**, and then specify your secret key value.  Once the variables are configured, you'll see the URL for accessing the application.
-
-![AWS Tracking Application](images/AWT10.png)
+## Run the application
 
-**Note:** If you don't know how to set variables, see [Environment properties and other software settings](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/environments-cfg-softwaresettings.html).
+Using the IntelliJ IDE, you can run your application. The first time you run the Spring Boot application, click the run icon in the Spring Boot main class, as shown in this illustration. 
 
-To access the application, open your browser and enter the URL for your application. You will see the login page for your application.
+![AWS Tracking Application](images/runapp.png)
 
-![AWS Blog Application](images/AWT11.png)
+**Note**: You can deploy this Spring Boot application by using AWS Elastic Beanstalk. If you do deploy this application to AWS Elastic Beanstalk, you need to set up an additional inbound rule. For information about deploying a web application, see the following document [Creating your first AWS Java web application](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/usecases/creating_first_project).
 
 ### Next steps
 Congratulations, you have created and deployed a Spring Boot application that interacts with Amazon RDS (and other AWS services). As stated at the beginning of this tutorial, be sure to terminate all of the resources you create while going through this tutorial to ensure that you’re no longer charged.
```

Comment:
As stated earlier, be sure to delete all of the resources that you created during this tutorial so that you won't continue to be charged.